### PR TITLE
Issue/2180

### LINKED ISSUE #2180: Add inline notice support to Give_Notices
## Issue Overview

Currently, We can only register admin notices with `Give_Notices` which will appear on top of any admin page. It will help developers if we will add inline notice support to this class.

For ref: https://github.com/WordImpress/Give/issues/2176

## Expected Behavior

Able to generate the inline messages.

## Current Behavior

we can only able to register notice which will render by the core on top of admin pages.

## Todos
- [ ] Tests
- [ ] Documentation

### PR DESCRIPTION
**Pull request by mehul0810**
## Description
This PR will resolve #2180 

## How Has This Been Tested?
I have tested this creating an inline notice for System Info.

## Screenshots (jpeg or gifs if applicable):
image

## Types of changes
New feature (non-breaking change which adds functionality) 

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/class-notices.php [hunk 1]
```diff
@@ -631,4 +631,48 @@ public static function print_frontend_notice( $message, $echo = true, $notice_ty
...
+	 * @param array  $notice_args An array of notice arguments.
+	 *
+	 * @todo Implement render_admin_notices function within this function in future.
+	 *
+	 * @access public
+	 * @since  1.8.17
+	 *
+	 * @return string
+	 */
+	public function print_admin_notices( $message, $notice_args = array() ) {
```
Comment (ravinderk) [inline includes/class-notices.php]: @mehul0810 Add PHPUnit test for this function

File: includes/class-notices.php [hunk 2]
```diff
@@ -631,4 +631,48 @@ public static function print_frontend_notice( $message, $echo = true, $notice_ty
...
+		$css_class = 'notice-' . $notice_args['notice_type'] . ' give-notice notice inline ';
+		$css_class .= ( $notice_args['dismissible'] ) ? 'is-dismissible' : '';
+		$output    .= sprintf(
+			'<div id="%1$s" class="%2$s">%3$s</div>' . " \n",
+			$css_id,
+			$css_class,
+			$message
+		);
+
+		if( ! $notice_args['echo'] ) {
```
Comment (raftaar1191) [inline includes/class-notices.php]: Please update WordPress Coding Standard

Comment (mehul0810) [reply includes/class-notices.php]: Fixed!

File: includes/class-notices.php (hunk 1 above)
Comment (mehul0810) [reply includes/class-notices.php]: @ravinderk Added Unit Tests

**Comment by ravinderk** [comment]
@mehul0810 Check my commit in this PR.

**Pull request merged**